test/openAPI/features/callback_email.feature

```diff
@@ -0,0 +1,109 @@
+        Then The response from the /callback/email endpoint is received
+        And The /callback/email response should be returned in a timely manner 15000ms
+        And The /callback/email response should have status 200
+
+
+    @smoke @unit @positive
+    Scenario: Callback endpoint successfully responds to a batch of emails sent via Messaging BB smoke type test
+
+        Given Batch of emails is sent and returns valid requestUID
+        When "POST" request with given "abcdef12345" as api_key and valid requestUID in the headers is sent
+        And The request contains "Meta-info" as meta and "Q2xpZW50LXNwZWNpZmljIGNvbnRlbnQsIG1vc3QgbGlrZWx5IEpTT04=" as content in the payload
+        Then The response from the /callback/email endpoint is received
+        And The /callback/email response should be returned in a timely manner 15000ms
+        And The /callback/email response should have status 200
+
+
+    @unit @positive
+    Scenario Outline: Callback endpoint successfully responds to an email sent via Messaging BB
+
+        Given Single email is sent and returns valid requestUID
+        When "POST" request with given "abcdef12345" as api_key and valid requestUID in the headers is sent
+        And The request contains "Meta-info" as meta and "<content>" as content in the payload
+        Then The response from the /callback/email endpoint is received
+        And The /callback/email response should be returned in a timely manner 15000ms
+        And The /callback/email response should have status 200
+
+        Examples:
+        | content                                                                                                                                                                                                                                                  | 
+        | ewogICJyZWNpcGllbnRJbmZvIjogewogICAgImVtYWlsIjogImN1c3RvbWVyQGV4YW1wbGUuY29tIiwKICAgICJlbWFpbENDIjogImN1c3RvbWVyLWNjQGV4YW1wbGUuY29tIiwKICAgICJlbWFpbEJDQyI6ICJjdXN0b21lci1iY2NAZXhhbXBsZS5jb20iLAogICAgIm5hbWUiOiAiRXhhbXBsZSBDdXN0b21lciIKICB9Cn0=     |
+        | ewogICJyZWNpcGllbnRJbmZvIjogewogICAgImVtYWlsIjogImN1c3RvbWVyMkBleGFtcGxlLmNvbSIsCiAgICAiZW1haWxDQyI6ICJjdXN0b21lcjItY2NAZXhhbXBsZS5jb20iLAogICAgImVtYWlsQkNDIjogImN1c3RvbWVyMi1iY2NAZXhhbXBsZS5jb20iLAogICAgIm5hbWUiOiAiRXhhbXBsZSBDdXN0b21lciIKICB9Cn0= |
+
+        # First:
+        # {
+        #   "recipientInfo": {
+        #   "email": "customer@example.com",
+        #   "emailCC": "customer-cc@example.com",
+        #   "emailBCC": "customer-bcc@example.com",
+        #   "name": "Example Customer"
+        #   }
+        # }
+        # Second:
+        # {
+        #   "recipientInfo": {
+        #   "email": "customer2@example.com",
+        #   "emailCC": "customer2-cc@example.com",
+        #   "emailBCC": "customer2-bcc@example.com",
+        #   "name": "Example Customer"
+        #   }
+        # }
```

Comment:
Should it stay here?

Reply:
I personally would leave it there to be sure what is this b64'ed code in Examples table.  What are your thoughts regarding it @KarolinaKopacz . Remove or leave?

Reply:
Ah ok, this is why you leave it here. It was confusing for me. I propose to remove it, for a developer is easy to decode it. 
I just curious how it will print on the testing web app, but this is something what we need to handle in web app, not here ;)

Reply:
removed